ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/data/Model.kt

```diff
@@ -1,12 +1,89 @@
 package ai.nnstreamer.ml.inference.offloading.data
 
+import androidx.room.ColumnInfo
 import androidx.room.Entity
 import androidx.room.PrimaryKey
+import org.json.JSONObject
+import java.io.File
 
 @Entity(tableName = "models")
 data class Model(
-    @PrimaryKey(autoGenerate = true)
+    @PrimaryKey
     val uid: Int = 0,
-    // TODO: Fill in the information based on pre-defined config file
     val name: String,
-)
+) {
+    constructor(
+        uid: Int,
+        name: String,
+        jsonObject: JSONObject = JSONObject(),
+        optionalJsonObject: JSONObject = JSONObject()
+    ) : this(uid, name) {
+        // TODO: The following is the parser for "single", which is a legacy conf
+        runCatching {
+            framework = jsonObject.getString("framework")
+        }.exceptionOrNull()
+
+        runCatching {
+            val modelArray = jsonObject.getJSONArray("model")
```

Comment:
It may not be directly related to this PR, but does nnstreamer single api support using multiple models at once?

Reply:
Do you mean the case when the model files are separated?

Reply:
Yes. I'm confused if one "single" conf file could have multiple model files

Reply:
@niley7464 The above 'single' is used slightly differently from the 'single' of the ```single``` API. BTW, isn't the ```single``` API a simple wrapper of ```tensor_filter```? If so, the API also supports multiple files (of a model)

Reply:
Oh I forgot that there was an `ml_single_open_with_option`

> The above 'single' is used slightly differently from the 'single' of the single API

Thank you for the explanation.